### PR TITLE
Formfield variant for PinInputTextField

### PR DESCRIPTION
**Pull request by alyyasser**
related issue: #13 

@TinoGuo , here's the PR. i've made some changes in example, and suply example for the new PinInputTextFormField too.

can you review the changes and example first? im afraid, im not testing and reviewing it enough to detect any breaking changes.

i'll wait for your feedback on this.

thank you.

### CODE REVIEW

File: lib/pin_input_text_field.dart [hunk 1]
```diff
@@ -21,11 +21,24 @@ abstract class PinDecoration {
 
   final ObscureStyle obscureStyle;
 
+  final String errorText;
```
Comment (TinoGuo) [inline lib/pin_input_text_field.dart:26]: Please add a comment for these variable.

**Review by TinoGuo — CHANGES_REQUESTED** [review]
Everything is perfect ✨, please just add some comments on the variables and help others to understand the source.

File: lib/pin_input_text_field.dart (hunk 1 above)
Comment (alyyasser) [reply lib/pin_input_text_field.dart:26]: ok, will do that today.

**Comment by alyyasser** [comment]
im done with the comments, if im missing some or there's another place that need to be explained, im open for another feedback.

**Review by TinoGuo — APPROVED** [review]
Excellent

**Pull request merged**